src/i18n/lang-fr.ts

```diff
@@ -0,0 +1,49 @@
+import { Translation } from "../models/Translation.interface";
+
+export const en: Translation = {
+    "themeInstalled": "Material Icon Theme a été installé.",
+    "howToActivate": "Comment activer les icônes",
+    "activate": "Activer",
+    "activated": "Material Icon Theme est actif.",
+    "neverShowAgain": "Ne plus afficher",
+    "themeUpdated": "Material Icon Theme a été mis à jour.",
+    "readChangelog": "Lire la liste des changements",
+    "angular": {
+        "alreadyEnabled": "Les icônes Angular sont déjà actifs.",
+        "alreadyDisabled": "Les icônes Angular sont déjà inactifs.",
+        "enableIcons": "Activer les icônes Angular",
+        "disableIcons": "Désactifer les icônes Angular",
+        "toggleIcons": "Basculer les icônes Angular"
+    },
+    "folders": {
+        "alreadyEnabled": "Les icônes de dossiers sont déjà actifs.",
+        "alreadyDisabled": "Les icônes de dossiers sont déjà inatifs.",
+        "enableIcons": "Activer les icônes de dossiers",
+        "disableIcons": "Déactiver les icônes de dossiers",
+        "toggleIcons": "Basculer les icônes de dossiers",
+        "specific": {
+            "name": "Specifique",
+            "description": "Icônes de dossiers spéciviques"
+        },
+        "classic": {
+            "name": "Classique",
+            "description": "Icônes de dossiers grises"
+        },
+        "blue": {
+            "name": "Bleu",
+            "description": "Icônes de dossiers bleues"
+        },
+        "none": {
+            "name": "Aucun",
+            "description": "Aucune icônes de dossiers"
+        }
+    },
+    "toggleSwitch": {
+        "on": "ON",
+        "off": "OFF"
+    },
+    "confirmReload": "Veullez redémarrer VS Code pour activer les icônes",
+    "reload": "Redémarer",
```

Comment:
I think "Redémarer" has a spelling mistake. So I think it should be written like in the line above as "Redémarrer". Please also check the spelling of the other words. I cannot merge the translation until there are some spelling mistakes in it.